vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -126,12 +135,21 @@ def _optimise_fig_layout_for_dashboard(fig):
                 # Reduce `margin_t` if not explicitly set.
                 fig.update_layout(margin_t=64)
 
-            if fig.layout.title.pad.t is None and "<br>" not in fig.layout.title.text:
-                # Reduce `title_pad_t` if title doesn't have subtitle and `title_pad_t` is not explicitly set.
-                fig.update_layout(title_pad_t=7)
-
         return fig
 
+    @_log_call
+    def pre_build(self):
+        try:
+            self.figure["title"]
+        except KeyError:
+            pass
+        else:
+            warnings.warn(
+                "Using `fig.layout.title` in your Plotly chart may cause misalignment with other "
+                "component titles on the screen. To ensure consistent alignment, consider using "
+                "`Graph.title` instead."
+            )
+
```

Comment:
I like that!

For context:

<img width="834" alt="Screenshot 2024-09-04 at 12 44 31" src="https://github.com/user-attachments/assets/608556e3-119d-449c-878d-3e53060d6518">

Reply:
Using the triple quotes actually makes it a bit more difficult to read (as an output), because the formatting seems off then 🤔

**Triple quotes**
![Screenshot 2024-09-04 at 14 18 48](https://github.com/user-attachments/assets/7d29f982-8a2b-4885-9e05-58df8d1caf48)

**Single quotes**
![Screenshot 2024-09-04 at 14 22 33](https://github.com/user-attachments/assets/3aa3523c-cd7c-4a76-9ad5-7e0acb463058)

Took me forever to figure out why the unit tests were failing with this change 😄 But if you use parentheses, you have to change the syntax of match statement: https://github.com/pytest-dev/pytest/issues/10595

Reply:
Ah weird, did you properly dedent (either manually, by putting the text to the very left), or by using
```py
from textwrap import dedent
print(dedent("""
	Text for which you want to remove
    leading whitespace when printed
    """))
```

Very nice tool :P